src/app/api/mock/hibp/data/fakeAllBreaches.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
I think providing this mock data is fine for now, if you want to (or we could consider in the future) generating fake data with the right types e.g. the way https://github.com/mozilla/blurts-server/blob/974957027a73df5f0a3091e8d24a46282dd02e6b/src/apiMocks/mockData.ts works, which would provide a little more fuzziness.

Reply:
I realize this makes testing more complicated, no need to do anything here but something to think about.

Reply:
This sounds great! I looked over the code, I think a future integration with the existing methods there will be very beneficial.